test/vars/MavenSpec.groovy

```diff
@@ -101,4 +101,39 @@ class MavenSpec extends JenkinsPipelineSpecification {
         then:
         1 * getPipelineMock("sh")('mvn -B -s settingsFileId -fae clean install')
     }
+
+        def "[maven.groovy] run mvn versions set"() {
+        setup:
+            def String newVersion = '3.6.2'
```

Comment:
just make sure that this version is aligned with other pieces (runtimes, apps, examples)